### ISSUE TITLE
Won't Load

### ISSUE THREAD

**Issue by c411um1906** [yawut/ntrview-wiiu#6]
When I load this up from the homebrew menu it just won't load. I have tried version 4 and 5 and they both crash when loading

**Comment by TVC202** [comment]
I'm having the same problem. I'm using tiramisu HB and have the files stored on my sd card for the front of the Wii U. Everytime I try to load ntrview from homebrew launcher, my system freezes.

**Comment by vgmoose** [comment]
@TVC202 can you provide this info?
> What version of HBL are you using (bottom-right) and what exploit/website do you use to load it?

Older versions of Homebrew Launcher / exploits may have issues with newer rpx files, latest is here: https://github.com/dimok789/homebrew_launcher/releases/tag/v1.5

**Comment by QuarkTheAwesome** [comment]
Yeah, see if updating HBL fixes it, if not please specify exactly what you see on screen during the freeze. Black screen? the HBL progress bar? ntrview's purple background? It's difficult to even begin to work out what the issue could be without that kind of detail.

It may also be worth checking `/vol/system_slc/logs` (`/system_slc/logs` in FTPiiU Everywhere) and seeing if any of the files in there start with the date of the crash, sometimes that can be enough for me to work with too.

**Comment by TVC202** [comment]
My bad. The system just freezes and stays on HBL when trying to launch ntrview. Nothing else happens.

I used wiiuexploit.xyz to install my homebrew and it's HB launcher v2.1 by dimok.

**Comment by TVC202** [comment]
I'm gonna update my HBL and see if things change. I'll post my results.
